babel.config.js

```diff
@@ -0,0 +1,29 @@
+module.exports = function(api) {
+
+  api.cache(true)
+
+  return {
+    presets: [
+      [
+        "@babel/env",
+        {
+          modules: false,
+          loose: true,
+          targets: {
+            browsers: ["last 1 version"]
+          },
+        }
+      ],
+      "@babel/react",
+    ],
+    plugins: [
+      "@babel/plugin-proposal-class-properties",
```

Comment:
Looks like only "@babel/plugin-proposal-class-properties" needed

Reply:
@HelloPopartz, what are you think?

Reply:
Yes! I'll remove it without further ado. I added those changes because the previous library was deprecated and in the migration guide where recommended